content/en/docs/release.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Release schedule and support policy"
+description: "This document describes the Notary Project release schedule and support policy"
+type: docs
+weight: 10
+---
+
+Since the Notary Project v1.0.0 release, Notary Project releases are scheduled quarterly.
+
+## Release versioning
+
+The Notary Project release version is in the form of `vX.Y.Z`, where `X` is the major version, `Y` is the minor version and `Z` is the patch version, following [Semantic Versioning](https://semver.org/spec/v2.0.0.html) terminology.
+
+## Patch release
+
+Patch releases provide users with bug fixes and security fixes. They do not contain new features. A patch release can be done by each Notary Project repository individually every quarter in a calendar year. A patch release to fix critical issues or security issues can be done on demand. A patch release can be cancelled if there is no new fixes since the previous release.
```

Comment:
Release has a cost associated with it and maybe 4 months is more reasonable than 3. But that's something that @notaryproject/notaryproject-org-maintainers  should weigh in on.

Reply:
Quarter-based release cadence sounds reasonable to me since the Notary Project is still in a fast iteration period. The Kubernetes community kept a quarter-based cadence for several years and changed the release cadence from four releases a year (once a quarter) to three releases a year. Maybe we can consider a 4-month release cycle after v2.

Reply:
Considering we still have many backlog issues not well addressed, so I would suggest quarterly based releases.